assets/js/components/Work/Tabs/Structure.jsx

```diff
@@ -2,74 +2,190 @@ import React, { useContext } from "react";
+          {work.fileSets.map(({ id, accessionNumber, metadata }) => (
+            <article key={id} className="media">
+              <figure className="media-left">
+                <p className="image is-64x64">
+                  <img
+                    src={`${iiifServerUrl}${id}/square/500,500/0/default.jpg`}
+                    placeholder="Fileset Image"
+                    onError={e => {
+                      e.target.src = "/images/1280x960.png";
+                    }}
+                  />
                 </p>
+              </figure>
+              <div className="media-content">
+                <div className="content">
+                  {isEditing ? (
+                    <>
+                      <div className="field">
+                        <strong>{accessionNumber}</strong>
+                      </div>
+                      <div className="field">
+                        <div class="field is-horizontal">
+                          <div class="field-label is-normal">
+                            <label class="label">Label</label>
+                          </div>
+                          <div class="field-body">
+                            <div class="field">
+                              <p class="control">
+                                <input
+                                  ref={register({ required: true })}
+                                  name={`label-${id}`}
+                                  data-testid="label"
+                                  className={`input ${
+                                    errors[`label-${id}`] ? "is-danger" : ""
+                                  }`}
+                                  type="text"
+                                  placeholder="Label"
+                                  defaultValue={metadata.label}
+                                />
+                              </p>
+                              {errors[`label-${id}`] && (
+                                <p className="help is-danger">
+                                  Label is required
+                                </p>
+                              )}
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+
+                      <div className="field">
+                        <div class="field is-horizontal">
+                          <div class="field-label is-normal">
+                            <label class="label">Description</label>
+                          </div>
+                          <div class="field-body">
+                            <div class="field">
+                              <p class="control">
+                                <textarea
+                                  ref={register({ required: true })}
+                                  name={`metadataDescription-${id}`}
+                                  data-testid="metadataDescription"
```

Comment:
I made a comment in another PR, and what I'd like to do for consistency in our code is handle `data-testid`s as follows:

`data-testid="[element type]-[name of thing separated by dashes]`

So, this test id would change to:
`data-testid="metadataDescription"` becomes `data-testid="textarea-metadata-description"`

And apply this to all `data-testid`s in this form please.